packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx

```diff
@@ -123,6 +123,35 @@ export const PromotionalTemplate = () => html`<pharos-layout style="margin: 1rem
   </div>
 </pharos-layout>`;
 
+export const Selectable = () => html`<pharos-layout tag="ol" style="margin: 1rem 0">
```

Comment:
I will trim this file down before merge. I wanted to temporarily create all combinations of props/variants to prove it was working.